windows-driver-docs-pr/install/inf-addservice-directive.md

```diff
@@ -124,6 +127,7 @@ ServiceBinary=path-to-service
 [AddReg=add-registry-section[, add-registry-section] ...]
 [DelReg=del-registry-section[, del-registry-section] ...]
 [BitReg=bit-registry-section[,bit-registry-section] ...]
+[BootFlags=value] (Windows 11 build Version <TBD> and later versions of Windows) 
```

Comment:
move this to the end since it is the newest (makes the parentheticals about what OS versions it is supported in go in order)